themes/default/static/programs/aws-iam-role-policyattachment-managedpolicy-java/src/main/java/myproject/App.java

```diff
@@ -0,0 +1,37 @@
+package myproject;
+
+import com.pulumi.Context;
+import com.pulumi.Pulumi;
+import com.pulumi.aws.iam.Role;
+import com.pulumi.aws.iam.RoleArgs;
+import com.pulumi.aws.iam.RolePolicyAttachment;
+import com.pulumi.aws.iam.RolePolicyAttachmentArgs;
+import static com.pulumi.codegen.internal.Serialization.*;
+
+public class App {
+    public static void main(String[] args) {
+        Pulumi.run(App::stack);
+    }
+
+    public static void stack(Context ctx) {
+        var role = new Role("my-role", RoleArgs.builder()        
+            .assumeRolePolicy(serializeJson(
+                jsonObject(
+                    jsonProperty("Version", "2012-10-17"),
+                    jsonProperty("Statement", jsonArray(jsonObject(
+                        jsonProperty("Action", "sts:AssumeRole"),
+                        jsonProperty("Effect", "Allow"),
+                        jsonProperty("Sid", ""),
+                        jsonProperty("Principal", jsonObject(
+                            jsonProperty("Service", "ec2.amazonaws.com")
+                        ))
+                    )))
+                )))
+            .build());
+
+        new RolePolicyAttachment("my-rpa", RolePolicyAttachmentArgs.builder()        
+            .role(role.name())
+            .policyArn(Output.format(ManagedPolicy.ReadOnlyAccess.getValue())))
```

Comment:
This fails as well: 

![image](https://github.com/pulumi/pulumi-hugo/assets/274700/800f42ca-bb01-44be-98a4-03491d1653e9)

This is because you need to add these imports:

```java
import com.pulumi.aws.iam.enums.ManagedPolicy;
import com.pulumi.core.Output;
```